src/pages/_app.tsx

```diff
@@ -24,6 +24,17 @@ class MyApp extends App<MyAppProps & ReduxWrapperAppProps<RootState>> {
     return { pageProps };
   }
 
+  componentDidMount() {
+    const tracker = window.document.createElement('script');
+    const firstScript = window.document.getElementsByTagName('script')[0];
+    tracker.defer = true;
+    tracker.setAttribute('site', `MVHUUREV`);
+    tracker.setAttribute('spa', 'auto');
+    tracker.src = 'https://cdn.usefathom.com/script.js';
+
+    firstScript.parentNode!.insertBefore(tracker, firstScript);
+  }
+
```

Comment:
I'd suggest we put this into this file: [app-init.tsx](https://github.com/blockstack/explorer/blob/feat/contract-call-debug/src/components/app-init.tsx), in a useEffect() wrapper.

Reply:
You'll also need to add in `typeof window !== 'undefined'` checks

Reply:
one more: please make it so this only runs in production

Reply:
👍